### PR TITLE
UGENE-6902. Cleanup Undo/Redo handling code in MSA Editor

### PR DESCRIPTION
**Pull request by yalgaer**

### CODE REVIEW

File: src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp [hunk 1]
```diff
@@ -57,65 +58,55 @@ MSAGeneralTab::MSAGeneralTab(MSAEditor *_msa)
...
 
-void MSAGeneralTab::sl_copyFormatStatusChanged(bool enabled) {
-    copyButton->setEnabled(enabled);
+void MSAGeneralTab::sl_copyFormatSelectionChanged(int index) {
+    QString formatId = copyType->itemData(index).toString();
+    msaEditor->getUI()->getSequenceArea()->sl_changeCopyFormat(formatId);
 }
 
 void MSAGeneralTab::connectSignals() {
+    MSAEditorSequenceArea *sequenceArea = msaEditor->getUI()->getSequenceArea();
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp:84]: Add `SAFE_POINT(sequenceArea != nullptr);`

File: src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp [hunk 2]
```diff
@@ -57,65 +58,55 @@ MSAGeneralTab::MSAGeneralTab(MSAEditor *_msa)
     initializeParameters();
     connectSignals();
 
-    copyButton->setEnabled(msa->getUI()->getCopyFormattedSelectionAction()->isEnabled());
-
     U2WidgetStateStorage::restoreWidgetState(savableTab);
 
-#ifdef Q_OS_MAC
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp:64]: Why did you remove tooltips?

**Comment by DmitriiSukhomlinov** [comment]
Minor comments, approved

**Review by DmitriiSukhomlinov — APPROVED** [review]

File: src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp (hunk 2 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp:64]: In line 108 I copy a  tooltip from the original action

File: src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_msa/General/MSAGeneralTab.cpp:84]: OptionsPanel is always initialized after main MAEditor->ui & sequence area is a required (non-null widget) which is initialized in the constructor of the MAEditor->ui

**Review by atiunov — APPROVED** [review]
